script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -226,22 +250,28 @@ This is a special weapon, created a long time ago, that has been restored for us
 [sha:ac511d468c54d0ade104d6b1a92c21e928ea6cd7]{
 -- Page 114, Offset 22430.
 -- すごく悪夢に見えます、と言いたいのをぐっっっと堪えて、大型の銃に見えます、と答えた。
-I try %{i}really%{/i} hard to keep myself from saying that it looks like a nightmare. I instead say that it looks like a large gun.
+I bite down painfully hard on my tongue to stop myself from saying that it looks like an absolute nightmare.
+ Instead, I say that it looks like a large gun.
```

Comment:
just say

not "just" as in "only, without elaboration" but rather "only, in contrast to what i was going to say before"